webapp/channels/src/utils/utils.tsx

```diff
@@ -1657,3 +1657,7 @@ export function sortUsersAndGroups(a: UserProfile | Group, b: UserProfile | Grou
 
     return aSortString.localeCompare(bSortString);
 }
+
+export function doesCookieContainsMMUserId() {
+    return document.cookie.includes('MMUSERID=');
+}
```

Comment:
Do you think we're likely to use this function elsewhere? I like giving it a name to make its purpose clearer, but I think it'd be better to leave it next to the code that uses it until we find somewhere else that wants to use it

Reply:
yes used in another place where this was used.